### DISCUSSION TITLE
Menu options and field titles not displaying properly, "An unexpected error occured"

### DISCUSSION THREAD

**Discussion by eksyh** [requarks/wiki#5339] [General]
Hello all,

We recently installed wiki.js and we're experiencing these issues on a fresh installation. All buttons, titles, fields, etc, appear to show their javascript placeholder names instead of the actual values. For example, dashboard.title instead of Dashboard, users.title instead of Users, etc. In addition to this, when first clicking any option in the admin console you get a red banner error message at the top indicating "An unexpected error occurred", but the nature of the error doesn't seem to be logging anywhere that wiki.js logs to on the system so we can't check to see what the actual issue is. Please take a look at the attached screenshot for clarification. Has anyone else encountered this issue and if so can you please provide guidance on how to go about addressing it? We have attempted to research this but the error message is very vague and doesn't yield valuable results in search. This installation is running directly on Linux, not within a Docker container. The OS is RHEL 8.5. Your help is very much appreciated, thank you!
![errors](https://user-images.githubusercontent.com/106356387/170599432-ce91cc1e-57cc-4428-a70b-511a5d022f8b.png)

**Comment by eksyh** [comment]
Bump, still trying to get this to work. Has anyone encountered this?

**Comment by tfboy** [comment]
Does this instance have a direct access to the Internet or is it firewalled off the Internet?
I ask because I have the same issue on a fresh install in an isolated environment and wondered if it was because upon install, it tries to get some updates but can't and so just provides these placeholder names with the red "unexpected error" message.

**Reply by eksyh** [reply]
We're running in an isolated environment as well, it doesn't have internet access. We had the same thought and ran a tcpdump on service start to see if it was reaching out to anything and it doesn't look like it. Unfortunately we're looking for alternatives as there doesn't seem to be any support for this.

**Reply by tfboy** [reply]
I was able to fix this. The issue is regarding the localisation files (language packs) which aren't included as standard and an attempt is made to load them up during the install process.

I have worked around this using the sideloading method. It does require creating a few docker configuration files, but it works fine and I was able to get rid of the message and have all the correct variable names shown in the language defined in the settings post-install.

See here for the couple of workarounds: https://stackoverflow.com/questions/62383153/wiki-js-cant-go-through-corporate-proxy

**Reply by eksyh** [reply]
Looks promising, thanks! I'll give that a shot.

**Comment by FeroxJB** [comment]
I also am having this issue.  However, I can't seem to figure out what it's pointing to and unable to load.

**Reply by FeroxJB** [reply]
I resolved the issue by correctly setting the sideload configuration based on this: https://docs.requarks.io/install/sideload

Embarrassingly I realized my mistake. Not paying attention, I did not save the files correctly from the github portal.   When I "Saved As", it downloaded the html link file.  lol   .. Just view the files as raw and hit Ctrl+S to save the files (At least with Edge/Chrome).  Dropped them in the C:\wiki\data\sideload folder and restarted the server and it was resolved.

**Comment by tertek** [comment]
If you are installing wiki on your local environment with Docker, please follow these instructions:
https://stackoverflow.com/a/64728579/3127170

It is also required to set `offline:true` within `config.yml`. This is missing in the official Documentation and should be added!

**Reply by tertek** [reply]
I have created a PR to the docs: https://github.com/requarks/wiki-docs/pull/59
